### DISCUSSION TITLE
How to get started with Python client.

### DISCUSSION THREAD

**Discussion by utkarshpandey12** [speechly/speechly#179] [Q&A] [answered]
I'm a beginer. The code in the python github repo seems overwhelming to me. I know python and want to pass and audio to speechly SLU API to get transcription results. Documentation says about using the login identity api --- > Then generating the access token and then using the SLU api to begin with tgranscription process. But in the repo I am unable to find these functions which are discussed in the code examples given in Python Client Repo Readme. Please suggest how to get started with the python client for my use case ..

Thanks
Utkarsh

**Comment by aukkonen** [comment, marked as answer]
@utkarshpandey12 
The gRPC API can indeed be a bit overwhelming for beginners. The functions you're talking about can be imported after installing the `speechly-api` package (e.g. `pip install speechly-api`).

I also added a [complete example](https://github.com/speechly/api/blob/master/python/speechly_grpc_example.py) that shows how to stream audio from a file to the API and read the returned results. I hope this helps you get ahead!

**Reply by utkarshpandey12** [reply]
@aukkonen 
This is exactly what I needed to get started. Thanks for adding this in the repo. Cheers : )

**Reply by utkarshpandey12** [reply]
@aukkonen    
Is the sampling rate = 16khz fixed or can I change it using some parameter ?
I have audio files sampled at 8khz currently I am converting them to 16khz using some online tool to test but my audio files are generated at 8khz sampling rate so is this a way in which I can configure sampling rate = 8khz

**Reply by utkarshpandey12** [reply]
@aukkonen Nevermind I think Only 16000 is supported
